criu/files-reg.c

```diff
@@ -1726,6 +2000,77 @@ static int rfi_remap(struct reg_file_info *rfi, int *level)
 	return 0;
 }
 
+/*
+ * Compares the file's build-id with the stored value.
+ * https://fedoraproject.org/wiki/Releases/FeatureBuildId#Unique_build_ID
```

Comment:
* I think we need a more detailed comment here. The link can be dead in a month.
* I see that this function can return -1, 0 or 1. Pls describe what each exit code means.